exporter/datadogexporter/config/config.go

```diff
@@ -256,6 +256,13 @@ type TracesConfig struct {
 	// If set to false the resource name will be filled with the instrumentation library name + span kind.
 	// The default value is `false`.
 	SpanNameAsResourceName bool `mapstructure:"span_name_as_resource_name"`
+
+	// SpanEventsAsTags defines the way to translate OpenTelemetry span events to datadog span tags.
+	// Possible values are:
+	// 	single: converts all the events to a single json object
+	//  multi: converts each span event to a separate tag.
+	//	The default value is `single`
+	SpanEventsAsTags string `mapstructure:"span_events_as_tags"`
```

Comment:
Instead of using a string, let's create a new type alias like we do here: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/7aac4a96a8ae1db4208f36472e4a60bf13884bb3/exporter/datadogexporter/config/config.go#L140-L166

Also, wondering if we should have this on a `traces::span_events` section, and call it... `tags_mapping`?